frontend/src/__tests__/cypress/cypress/utils/oc_commands/baseCommands.ts

```diff
@@ -51,3 +51,48 @@ export const patchOpenShiftResource = (
     return result;
   });
 };
+
+/**
+ * Wait for a specific pod to become ready across all namespaces.
+ *
+ * @param podNameContains A substring to partially match against the pod name (e.g., "jupyter-nb").
+ * @returns A Cypress chainable that waits for the pod to be ready.
+ * @param timeout The amount of time to wait for the pod to become ready (default is 10 seconds, e.g., '10s', '30m').
+ * @param namespace The namespace to search for the pod. Defaults to `-A` for all namespaces, or can specify a specific namespace with `-n <namespace>`.
+ */
+export const waitForPodReady = (
+  podNameContains: string,
+  timeout = '10s',
```

Comment:
Can you review the timeout here please? i ran the test against my psi cluster and the test failed. It timed out after 60000ms. 

![image](https://github.com/user-attachments/assets/91092851-12ca-4dcc-91f9-d7497564b133)

Reply:
@ConorOM1 Can you add `timeout: N * 1000` (after "failOnNonZeroExit: false" ) to explicitly wait for the N seconds needed ?

Reply:
@manosnoam For reusability I've handled it like so [Here](https://github.com/opendatahub-io/odh-dashboard/pull/3553/files#diff-b7629ad31b1c4e61dacfcab3dc67691ed74541f8453085aa9897aba311ad0993R73) We then modify the timeout as needed in the function call